src/main/java/net/sf/jabref/exporter/BibDatabaseWriter.java

```diff
@@ -289,7 +289,7 @@ private void writeMetaData(Writer out, MetaData metaData) throws IOException {
         // write groups if present. skip this if only the root node exists
         // (which is always the AllEntriesGroup).
         GroupTreeNode groupsRoot = metaData.getGroups();
-        if ((groupsRoot != null) && (groupsRoot.getChildCount() > 0)) {
+        if ((groupsRoot != null) && (groupsRoot.getNumberOfChildren() > 0)) {
```

Comment:
OK! You are probably right that isLeaf is harder to understand. Since at
least Collection and String use isEmpty it may be worthwhile. But then it
is not that crucial as one clearly can understand the current code
(obviously).

(Saw at least one more place where isEmpty/isLeaf can be used.)